gulpfile.js

```diff
@@ -69,7 +69,7 @@ gulp.task('fixup-swagger', function () {
   autorest.fixupRawSwagger('./swagger/bifrost.swagger.before.json', './swagger/bifrost.swagger.json');
 });
 
-gulp.task('autorest', ['clean-autorest', 'fixup-swagger'], function () {
```

Comment:
I have reverted that, but I'm confused: how do I generate new api from updated swagger file?
Some time ago, I was updating bifrost.swagger.json, and the calling gulp autorest. If I do it right now, my changes in bifrost.swagger.json are reverted.